orchestrator-salt/src/main/resources/salt/salt/fluent/agent-start.sls

```diff
@@ -0,0 +1,43 @@
+{%- from 'fluent/settings.sls' import fluent with context %}
+{% if fluent.enabled %}
+/etc/td-agent/pos:
+  file.directory:
+    - name: /etc/td-agent/pos
+    - user: "{{ fluent.user }}"
+    - group: "{{ fluent.group }}"
+    - mode: 740
+    - recurse:
+      - user
+      - group
+      - mode
+
+/etc/td-agent/td-agent.conf:
+  file.managed:
+    - source: salt://fluent/template/td-agent.conf.j2
+    - template: jinja
+    - user: "{{ fluent.user }}"
+    - group: "{{ fluent.group }}"
+    - file_mode: 640
+
+/etc/td-agent/output.conf:
+  file.managed:
+    - source: salt://fluent/template/output.conf.j2
+    - template: jinja
+    - user: "{{ fluent.user }}"
+    - group: "{{ fluent.group }}"
+    - file_mode: 640
+{%- if salt['cmd.retcode']('test -f /lib/systemd/system/td-agent.service') == 0 %}
+update_systemd_units:
```

Comment:
Systemctl files can be reloaded a bit more elegant way:

https://github.com/hortonworks/cloudbreak/blob/154f49a65ac17885a75a84a84ad5aa57f4e90927/orchestrator-salt/src/main/resources/salt/salt/gateway/init.sls#L226

Reply:
i had an additional fix as  `/lib/systemd/system/td-agent.service` should not be edited, so im copying td-agent.service into /etc/systemd/system before im updating it, therefore i keep the `test -f` command as i would need to check the file in multiple module calls. (also that checks systemd is used at all..so that is shorter)
anyway i have used the systemd reload and service start the same way as it was with he gateway example that you have showed

Reply:
I don't think this is a valid argument

Reply:
ok, then, it's updated